projects/configs/despiad.yaml

```diff
@@ -0,0 +1,39 @@
+#  Copyright (c) 2024 University College London Hospitals NHS Foundation Trust
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+project:
+    name: "despiad"
+    modalities:
+      - "CT"
+      - "PT"
+
+tag_operation_files:
+    base:
+        - "base.yaml"
+        - "ct.yaml"
+        - "pet.yaml"
+        - "despiad.yaml"
+    manufacturer_overrides: []
+
+allowed_manufacturers:
+  - regex: ".*"
+    exclude_series_numbers: []
```

Comment:
I think we should go through and find out all of the manufacturers before we allow the all through, as CT can have burned in data

Reply:
good point, we're now limiting to `^ge medical systems`